src/qtfirebaseremoteconfig.cpp

```diff
@@ -13,14 +15,22 @@ QtFirebaseRemoteConfig::QtFirebaseRemoteConfig(QObject *parent) :
     __appId(nullptr)
 {
     __QTFIREBASE_ID = QString().sprintf("%8p", this);
+
     if(self == 0)
     {
         self = this;
         qDebug() << self << "::QtFirebaseRemoteConfig" << "singleton";
     }
 
-    QTimer::singleShot(500, this, SLOT(beforeInit()));
-    connect(qFirebase,&QtFirebase::futureEvent, this, &QtFirebaseRemoteConfig::onFutureEvent);
+    if (PlatformUtils::googleServiceAvailable()) {
```

Comment:
Thanks. I'll pull it it. I'll add an issue to add this to the other modules depending on Google Services:
https://firebase.google.com/docs/cpp/setup#requiring_google_play_services